solutions/USACO jan16-radio-contact.mdx

```diff
@@ -25,6 +25,8 @@ Either one of the following happens:
 
 ## Implementation
 
+**Time Complexity:** $\mathcal{O}(N^2)$
```

Comment:
NM?

Reply:
woops forgot to change that when copy pasting 😭